src/components/NewGameItem.tsx

```diff
@@ -0,0 +1,62 @@
+import { Dispatch, FC, SetStateAction, useState} from 'react';
+import axios from 'axios';
+import { Button, Card, CardActions, CardContent, CardMedia, Typography } from '@mui/material';
+import { GameItemModal } from './GameItemForm';
+import { GameItem } from '../types/GameType';
+import { FieldValues } from 'react-hook-form';
+import { getFullGameInfo } from '../utils/getFullGameInfo';
+
+export const NewGameItem: FC<{item: GameItem, gameItems: GameItem[], setGameItems: Dispatch<SetStateAction<GameItem[]>>}> = ({item, gameItems, setGameItems}) => {
+  const [open, setOpen] = useState(false)
+
+  const handleDelete = (): void => {
+    axios.delete(`http://localhost:3005/api/games/${item.id}`).catch(err => console.error(err))
+    const newGameSet = gameItems.filter((game) => game.id !== item.id)
+    setGameItems(newGameSet)
+  }
+
+  const openEditModal = (): void => {
+    setOpen(true)
+  }
+
+  const handleSubmitEdit = async (data: FieldValues ): Promise<void> => {
+    const fullGame = getFullGameInfo(data)
```

Comment:
`fullGameInfo` instead of `fullGame`
Also minor spacing issue after `FieldValues`